exporter/exporterhelper/internal/base_exporter.go

```diff
@@ -210,7 +210,6 @@ func WithQueueBatch(cfg QueueConfig, set QueueBatchSettings[request.Request]) Op
 	return func(o *BaseExporter) error {
 		if !cfg.Enabled {
 			o.ExportFailureMessage += " Try enabling sending_queue to survive temporary failures."
-			return nil
```

Comment:
Why do we want to proceed here?

Reply:
Ok, made a different change. I was checking for the sizer even when disabled, and that would fail unless I am copying the config and settings all the time. But I changed now to do that only if config is enabled.